cloudmock/aws/mockec2/vpcs.go

```diff
@@ -127,11 +126,7 @@ func (m *MockEC2) DescribeVpcs(request *ec2.DescribeVpcsInput) (*ec2.DescribeVpc
 					}
 				}
 			default:
-				if strings.HasPrefix(*filter.Name, "tag:") {
```

Comment:
hasTag checks the filter values - we could probably rename it in another PR to `defaultFilters` or something